### ISSUE TITLE
RTL Support Doesn't Work

### ISSUE THREAD

**Issue by mfalthaw** [airbnb/rheostat#282]
Thanks for your contributions and for supporting RTL!

We were testing rheostat slider and we encountered issues with RTL. (see gif below)

![slider_rtl](https://user-images.githubusercontent.com/15677820/89868484-8bdb6500-dbbb-11ea-8dc6-c7216c96e355.gif)

Slider direction is flipped (opposite to the mouse movement direction)

relates to #235

**Comment by richardgreg** [comment]
Hi, I would like to work on this issue.

**Comment by richardgreg** [comment]
@ljharb is this repository still actively maintained and used?

**Comment by ljharb** [comment]
@richardgreg yes. You don't have to ask to work on an issue; anything with a "help wanted" label is up for grabs.

**Comment by a15n** [comment]
Hello @ljharb and @richardgreg I've fixed the RTL issue on localhost. It required quite a few changes like this within `<Slider/>` ....

![Screenshot 2023-01-10 at 5 57 39 PM](https://user-images.githubusercontent.com/7050871/211693544-e7c381db-65d9-4402-b794-23e354f09642.png)


I've verified that this works for each RTL Story. Unfortunately, when I try to `git push` to make a pull request I get this error....

```
remote: Permission to airbnb/rheostat.git denied to a15n.
fatal: unable to access 'https://github.com/airbnb/rheostat.git/': The requested URL returned error: 403
```


I'm looking into that now, but thought I'd revive this issue.

**Comment by ljharb** [comment]
I wouldn’t expect a push to work on https; you may need to try ssh - but either way, if what you have builds on #285, can you push a branch to your fork, and i can pull in your changes to that PR?

**Comment by a15n** [comment]
I've made this demo PR, which highlights that [RTL auto-flipping](https://github.com/airbnb/rheostat/pull/235#discussion_r309876723) is no longer happening. It hard-codes the correct left/right direction in the JS, but it cannot change the CSS. Regardless, this PR is just demonstrating that RTL would work again if the auto-flipping were working. 

@ljharb, @rcourtie, @majapw any idea what may have caused RTL auto flipping to stop working sometime after [it was introduced in 2019?](https://github.com/airbnb/rheostat/pull/235)
